Bottom: f70d2fc67c3ed1e8a66da3e39fd559f2404f1aae
Top:    8a1fd1583aa25621428c41159fd7c7081e1b8446
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 23:56:12 +0000

Subtract vias from ...


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 017fbf4..33fe6c3 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1295,6 +1295,10 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
        *
        */
 
+      fprintf (stderr, "Subtracting pin + via drills\n");
+      r_search (PCB->Data->pin_tree, &bounds, NULL, pv_drill_callback, &info);
+      r_search (PCB->Data->via_tree, &bounds, NULL, pv_drill_callback, &info);
+
       if (info.poly == NULL)
         {
           fprintf (stderr, "Skipping layer group %i, info.poly was NULL\n", group);
